src/parser/tests.rs

```diff
@@ -76,6 +76,43 @@ BEGIN { }
 map<i32, i32> count;
 count = 0;
 BEGIN { }
+    "#,
+    //function stuff
+    r#"
+    fn_name(i32 param) -> i32{}
+    BEGIN { }
+        "#,
+    r#"
+    fn_name() -> i32{
+        i = 0;
+    }
```

Comment:
Is it by default the function returns the last statement?

Reply:
Either that, or I need to require a return statement in a function block and make it different than others. Basically, was planning on making there an error if there's no return in the function block after tokenization, but that should be done during parser.

Reply:
I recommend making every function have a return statement.

Reply:
yeah I agree. Type checking the return value should be done at in verifier, but that there is one can be done in parsing